### DISCUSSION TITLE
enable_area_selection and callback function about return all selected point coordinate

### DISCUSSION THREAD

**Discussion by fire8flag** [pyvista/pyvista#4163] [Q&A] [answered]
I want to implement a box selection function: draw a rectangular area and select all the points inside and return the coordinates of all the points.How should I implement it?
Thank you

**Comment by fire8flag** [comment]
Reference：
https://kitware.github.io/vtk-examples/site/Cxx/Picking/AreaPicking/

**Comment by tkoyama010** [comment, marked as answer]
We already discuss in https://github.com/pyvista/pyvista/issues/4162 so mark this discussion as resolved.
